lib/intercom/api_operations/bulk/load_error_feed.rb

```diff
@@ -0,0 +1,13 @@
+module Intercom
+  module ApiOperations
+    module Bulk
+      module LoadErrorFeed
+        def load_error_feed(params)
+          response = @client.get("/jobs/#{params[:id]}/error", {})
```

Comment:
maybe `params.fetch(:id)` instead of `params[:id]` so we don't silently call `/jobs/nil/error`